src/react/src/components/DashboardUpdateFacilityLocation.jsx

```diff
@@ -86,7 +91,10 @@ function DashboardUpdateFacilityLocation({
                     fetching={targetFetching}
                     error={targetError}
                     handleEnterKeyPress={fetchTargetFacilityOnEnterKeyPress}
-                    title="Facility"
+                    title={dashboardFacilityCardTitles.facility}
+                    highlightBackground={dashboardFacilityMergeCards.includes(
```

Comment:
Don't pass highlightBackground if you don't need to change the background for the card in the DashboardDeleteFacility component.